src/main/resources/templates/fragment/script.html

```diff
@@ -10,6 +10,8 @@
     <script th:src="@{/webjars/bootstrap/4.3.1/js/bootstrap.min.js}"></script>
     <script th:src="@{/webjars/leaflet/1.4.0/leaflet.js}"></script>
     <script th:src="@{/js/turf.min.js}"></script>
+    <script th:src="@{/js/map.utils.js}"></script>
+    <script th:src="@{/js/validation.js}"></script>
```

Comment:
Are you sure this 2 scripts needed for all pages or just for index?

Reply:
Done.